crates/router/src/connector/klarna/transformers.rs

```diff
@@ -45,12 +45,14 @@ pub struct KlarnaPaymentsRequest {
     order_amount: i64,
     purchase_country: String,
     purchase_currency: enums::Currency,
+    merchant_reference1: Option<String>,
 }
 
 #[derive(Default, Debug, Deserialize)]
 pub struct KlarnaPaymentsResponse {
     order_id: String,
     fraud_status: KlarnaFraudStatus,
+    merchant_reference1: Option<String>,
```

Comment:
This issue is regarding connector payment request , please revert the response changes .

Reply:
Ok